doc/preview.yml

```
@@ -0,0 +1,57 @@
+site:
+  title: Neo4j Graph Data Science
+  url: https://neo4j.com/docs
+  start_page: graph-data-science:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ../../
+    start_path: public/doc
+    branches: 
+    - HEAD
+    include: public/doc/
+    exclude:
+    - '!**/_includes/*'
+    - '!**/readme.adoc'
+    - '!**/README.adoc'
+ui:
+  bundle:
+    url: https://s3-eu-west-1.amazonaws.com/static-content.neo4j.com/build/ui-bundle-latest.zip
+    snapshot: true
+  output_dir: /assets
+
+runtime:
+  fetch: true
+
+urls:
+  html_extension_style: indexify
+
+asciidoc:
+  extensions:
+  - "@neo4j-documentation/remote-include"
+  - "@neo4j-documentation/macros"
+  - "@neo4j-antora/antora-page-roles"
+  - "@neo4j-antora/antora-table-footnotes"
+  attributes:
+    page-theme: docs
+    page-type: Docs
+    page-search-type: Docs
+    page-search-site: Reference Docs
+    page-canonical-root: /docs
+    page-pagination: true
+    page-no-canonical: true
+    page-origin-private: true
```

```suggestion
    page-origin-private: false
```

As this is a public repo, when we publish the docs, by changing this attribute we can include an `Edit this page` link on each page, which links to the page's location in the repo.

---

Cool :) done